includes/abstracts/abstract-wc-order.php

```diff
@@ -272,7 +272,7 @@ public function update() {
 	 * Delete data from the database.
 	 * @since 2.7.0
 	 */
-	public function delete() {
+	public function delete( $force_delete = false ) {
 		wp_delete_post( $this->get_id() );
```

Comment:
Yup that's for later when each thing get's its store. I wanted to shut up the strict notices in the meantime (since I changed WC_Data).